### DISCUSSION TITLE
How to go about supporting a new OS

### DISCUSSION THREAD

**Discussion by nandbarkin** [DataDog/datadog-agent#11570] [Q&A]
Need to install datadog agent in an OS called Photon OS
The official page can be found here: https://vmware.github.io/photon/
What would it take to make the datadog agent work in this OS?

**Comment by sgnn7** [comment]
Hi @nandbarkin,
Given that Photon OS seems to come with Docker per-installed, you should be able to run the DD Agent in a container with mounted volumes for monitoring (see [here](https://docs.datadoghq.com/agent/docker/?tab=standard) for more details). Please let us know if you encounter any problems.
